src/controllers/api.controller.ts

```diff
@@ -430,33 +430,29 @@ export const claimReport = async (
         id: Number(claimId)
       }
     });
-    if (!claim) throw new createError.NotFound("Claim does not exist");
+    if (!claim) throw new createError.NotFound('Claim does not exist');
 
     const baseQuery = `
       SELECT DISTINCT
-          n1.name AS name,
-          n1.thumbnail AS thumbnail,
-          n1."nodeUri" AS link,
-          c.id AS claim_id,
-          c.statement AS statement,
-          c.stars AS stars,
-          c.score AS score,
-          c.amt AS amt,
-          c."effectiveDate" AS effective_date,
-          c."howKnown" AS how_known,
-          c.aspect AS aspect,
-          c.confidence AS confidence,
-          e.label AS claim,
-          e2.label AS basis,
-          n3.name AS source_name,
-          n3.thumbnail AS source_thumbnail,
```

Comment:
@0marSalah why are we removing thumbnail here?  btw it would be better not to change the indent and just change the real line, so we can see what meaningful change there is.  Is it because it is unused?